### ISSUE TITLE
Can't return to vim mode in atlassian.net JIRA

### ISSUE THREAD

**Issue by simonweil** [vimperator/vimperator-labs#298]
Hi, 
First of all, I love your this FireFox extension and I can't work without it,
So Thank You!

My only issue is that when in atlassian.net using JIRA,
once I leave vim mode by clicking on S-ESC it doesn't work to return to vim mode.

Is there any solution for this?

**Comment by devkev** [comment]
I find this happens occasionally, and that Jira is particularly prone to it.  While sometimes I'm unable to get back out of it, I often have success by clicking in an input field, and then hitting S-Esc again.  If the page has no input fields then I hit C-h to get the history sidebar, which gives focus to a search input field.

I think I've also seen the opposite, ie. where I have to click outside a text field before S-Esc will work again to get out of passthrough mode.

**Comment by gkatsev** [comment]
Yep, it's an issue. If you have an `<Insert>` key, you could use that to exit pass-through mode. It seems to be working fine.
We'll have to investigate why it isn't exiting correctly.

**Comment by dotancohen** [comment]
I can confirm the issue as well. Firefox 45 ESR, Vimperator 3.15, CentOS 7.2
